object_manager.go

```diff
@@ -562,6 +562,66 @@ func (objMgr *ObjectManager) DeleteCNAMERecord(ref string) (string, error) {
 	return objMgr.connector.DeleteObject(ref)
 }
 
+func (objMgr *ObjectManager) CreateTXTRecord(recordname string, text string, dnsview string) (*RecordTXT, error) {
+
+	recordTXT := NewRecordTXT(RecordTXT{
+		View: dnsview,
+		Name: recordname,
+		Text: text,
+	})
+
+	ref, err := objMgr.connector.CreateObject(recordTXT)
+	recordTXT.Ref = ref
+	return recordTXT, err
+}
+
+func (objMgr *ObjectManager) GetTXTRecordByRef(ref string) (*RecordTXT, error) {
+	recordTXT := NewRecordTXT(RecordTXT{})
+	err := objMgr.connector.GetObject(recordTXT, ref, &recordTXT)
+	return recordTXT, err
+}
+
+func (objMgr *ObjectManager) GetTXTRecord(name string) (*RecordTXT, error) {
+	if len(name) == 0 {
+		return nil, nil
+	}
+	var res []RecordTXT
+
+	recordTXT := NewRecordTXT(RecordTXT{Name: name})
+
+	err := objMgr.connector.GetObject(recordTXT, "", &res)
+
+	if err != nil || res == nil || len(res) == 0 {
+		return nil, err
+	}
+
+	return &res[0], nil
+}
+
+func (objMgr *ObjectManager) UpdateTXTRecord(recordname string, text string) (*RecordTXT, error) {
+	var res []RecordTXT
```

Comment:
```suggestion
	var res *RecordTXT
```
is it necessary for it be a slice ?

Reply:
@chinmayb thanks a lot, actually no need to. Updated.

Reply:
@chinmayb actually it is really required, see https://github.com/infobloxopen/infoblox-go-client/pull/89/commits/ba9fe18f76623d2336dedb9361e5c3f7bf7a5c8a .